server/routes/userRequest.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const router = express.Router();
+const UserRequest = require('../../db/UserRequest.js');
+
+//get a single user request
+router.get('/homes/:request', (req, res) => {
```

Comment:
I think it would be okay to refer to the id as follows
```suggestion
router.get('/homes/:home', (req, res) => {
```

In this route and all that follow

Reply:
Good idea!